apps/ios/Shared/Views/Chat/ChatItem/CIFileView.swift

```diff
@@ -45,7 +45,25 @@ struct CIFileView: View {
             .padding(.leading, 10)
             .padding(.trailing, 12)
         }
-        .disabled(file == nil || (file?.fileStatus != .rcvInvitation && file?.fileStatus != .rcvAccepted && file?.fileStatus != .rcvComplete))
+        .disabled(!itemInteractive)
+    }
+
+    var itemInteractive: Bool {
+        if let file = file {
+            switch (file.fileStatus) {
+            case .sndStored: return false
+            case .sndTransfer: return false
+            case .sndComplete: return false
+            case .sndCancelled: return false
+            case .rcvInvitation: return true
+            case .rcvAccepted: return true
+            case .rcvTransfer: return false
+            case .rcvComplete: return true
+            case .rcvCancelled: return false
```

Comment:
I think it's cleaner to just list cases when it is interactive

Reply:
I'd rather not at least for now, there will be new status(es) probably, this is just to make it compile and not forget to change later